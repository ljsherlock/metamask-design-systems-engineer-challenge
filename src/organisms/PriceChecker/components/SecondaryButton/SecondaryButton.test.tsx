import { shallow } from 'enzyme';
import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import TestComponent from '.'

describe('Secondary Button', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TestComponent />);
    expect(wrapper).toMatchSnapshot();
    // On the first run of this test, Jest will generate a snapshot file automatically.
  });
});