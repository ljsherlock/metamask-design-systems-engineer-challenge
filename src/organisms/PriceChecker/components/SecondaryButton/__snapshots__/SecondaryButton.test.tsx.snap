// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Secondary Button renders correctly 1`] = `ShallowWrapper {}`;
